api/src/main/java/io/strimzi/api/kafka/model/Storage.java

```diff
@@ -34,9 +35,11 @@
 
     public static final String TYPE_EPHEMERAL = "ephemeral";
     public static final String TYPE_PERSISTENT_CLAIM = "persistent-claim";
+    public static final String TYPE_JBOD = "jbod";
+
     private Map<String, Object> additionalProperties = new HashMap<>(0);
 
-    @Description("Storage type, must be either 'ephemeral' or 'persistent-claim'.")
+    @Description("Storage type, must be either 'ephemeral', 'persistent-claim' or 'jbod'.")
```

Comment:
I think Daniel usually wanted to have _'ephemeral', 'persistent-claim', or 'jbod'_ ... i.e. with additional `,`.